node/node.go

```diff
@@ -606,7 +607,7 @@ func (n *Node) loadSecurityConfig(ctx context.Context) (*ca.SecurityConfig, erro
 		// - We wait for CreateSecurityConfig to finish since we need a certificate to operate.
 
 		// Attempt to load certificate from disk
-		securityConfig, err = ca.LoadSecurityConfig(ctx, rootCA, krw)
+		securityConfig, err = ca.LoadSecurityConfig(ctx, rootCA, krw, n.config.ForceNewCluster)
```

Comment:
This is inside `if securityConfig == nil`. My understanding is that the second call is for the case where we didn't previously have the root CA, and we just either bootstrapped one or downloaded the cert.

As to the questions, I don't have a specific opinion on the behavior in this case.